examples/dump_planes.py

```diff
@@ -29,25 +29,11 @@ def dump(ome_tiff_fn):
     print "image size: %d x %d" % (W, H)
     print "pixel type:", dtype
     for i in xrange(img_count):
```

Comment:
Can some of the above values be removed now that they are included as part of the new open_array method? (H, W, dtype, interleaved) Or is it still useful to keep them for the purposes of the print statements?